src/server.h

```diff
@@ -116,6 +116,7 @@ struct ServerSoundParams
 	float max_hear_distance;
 	bool loop;
 	float fade;
+	float pitch;
```

Comment:
now we are using C++11 you can do "float pitch = 1.0f;"  directly, maybe convert the whole constructor in that style ? :)